kinde_fastapi/examples/management_token_example.py

```
@@ -0,0 +1,119 @@
+# Setup logging
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+# Initialize FastAPI app
+app = FastAPI(title="Kinde Management Token Example with Introspection")
+
+# Security scheme for bearer token
+security = HTTPBearer()
+
+# Extract, introspect, and validate management token from header
+def get_management_token(credentials: HTTPAuthorizationCredentials = Depends(security)) -> ManagementTokenManager:
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)